java/src/main/java/Main.java

```diff
@@ -0,0 +1,15 @@
+import search.BinarySearch;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
@ricktjwong I don't think we should be including entry points or test cases. The purpose of LYANL is to introduce the language through well known algorithms. Maybe in the future we can have proper project setups to teach the ecosystems and how to deploy (this is actually more useful than the language itself IMO). But for now we should stick to algorithm implementation only.